Convert telnetlib3-feedstock to v1 format and build last version

beenje opened the pull request:
This PR converts telnetlib3-feedstock to a v1 recipe and switch the conda build tool to rattler-build.
It has been automatically generated with [feedrattler v0.3.14](https://github.com/hadim/feedrattler).

Changes:
- [x] 📝 Converted `meta.yaml` to `recipe.yaml`
- [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
- [x] 🔧 Updated `conda-forge.yml` to use `rattler-build` and `pixi` (optional)
- [x] Bumped the version and reset the build number
- [x] 🐍 Applied temporary fixes for `python_min` and `python_version`
- [x] 🔄 Rerender the feedstock with conda-smithy
- [x] Ensured the license file is being packaged.
- [x] Added myself as maintainer

beenje committed:
Convert to v1 feedstock and use pixi as conda install tool

beenje committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.11.03.23.09.34

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.9

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ No valid build backend found for Python recipe for package `telnetlib3` using `pip`. Python recipes using `pip` need to explicitly specify a build backend in the `host` section. If your recipe has built with only `pip` in the `host` section in the past, you likely should add `setuptools` to the `host` section of your recipe.
 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `host` section of the recipe, you should usually use the pin python ${{ python_min }}.* for the `python` entry.
   - For the `run` section of the recipe, you should usually use the pin python >=${{ python_min }} for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19064742605](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19064742605). Examine the logs at this URL for more detail.</sub>

beenje committed:
Build for version 2.0.8

beenje committed:
Add Python-2.0 license

Used for copy of telnetlib

beenje committed:
Add beenje as maintainer

beenje committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.11.03.23.09.34

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.9

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

beenje commented:
@atrawog I see this feedstock hasn't been updated in a while.
I'm willing to be added as maintainer.

I updated the recipe to latest version and converted it to v1 format.

beenje commented:
@conda-forge/core, this feedstock seems abandoned.
The current maintainer hasn't replied to this PR.

I added myself as maintainer. Could you please review and merge?
